app/controllers/admin/users_controller.rb

```diff
@@ -1,10 +1,31 @@
 module Admin
   class UsersController < Admin::BaseController
+    before_action :set_user, only: [:edit]
+
     def index
       ransack_params = params[:q]
       @q = User.search(ransack_params)
       @users = @q.result(distinct: true).
                paginate(page: params[:page], per_page: 20)
     end
+
+    def edit; end
+
+    def update
+      params.permit! # It's OK because of admin
```

Comment:
__[Brakeman]__

Parameters should be whitelisted for mass assignment


<!-- Comment by [SideCI](https://sideci.com) -->